packages/analytics/get-product-intent-from-url/index.ts

```diff
@@ -0,0 +1,29 @@
+const products = [
+  'packer',
+  'terraform',
+  'vault',
+  'boundary',
+  'consul',
+  'nomad',
+  'waypoint',
+  'vagrant',
+] as const
+
+type Product = typeof products[number]
+
+export const getProductIntentFromURL = (url?: string): Product | null => {
```

Comment:
Can we add a docs comment here explaining what the function does?

Reply:
Yessssir! https://github.com/hashicorp/web-platform-packages/pull/63/commits/f805262171896cd58533b1b3f42f84e107248f75